tutorials/notebooks/example_keras_imagenet.ipynb

```diff
@@ -1,388 +1,486 @@
 {
- "cells": [
```

Comment:
Are the changes here automatically generated? because if it's manually then I need an explanation (and if it's not then please verify that they are indeed required)

Reply:
manually. Please see notebook in pycharm.